front/src/modules/trainschedule/components/DriverTrainScheduleV2/exportDriverScheduleCSV.ts

```diff
@@ -81,12 +71,14 @@ const overloadSteps = (
     lineCode: op.line_code,
     trackName: op.track_name,
   }));
-  const speedsAtSpeedLimitChange = speedLimits.map((speedLimit) => ({
-    position: mmToM(speedLimit.position),
-    speed: interpolateValue(trainRegime, speedLimit.position, 'speeds'),
-    speedLimit: speedLimit.speed,
-    time: interpolateValue(trainRegime, speedLimit.position, 'times'),
-  }));
+  const speedsAtSpeedLimitChange = speedLimits.speeds.map((_speed, index) => {
+    const boundary = index > 0 ? speedLimits.internal_boundaries[index - 1] : 0;
+    return {
+      position: boundary,
+      speed: interpolateValue(trainRegime, mToMm(boundary), 'speeds'),
```

Comment:
I'm a bit confused, but technically, boundary is already in mm (according to the description in `generatedEditoastApi`). So either the description is wrong, or the calculation is a bit weird here

Reply:
According to @Erashin, the description seems to be right.

Reply:
Ok,discussed with @RomainValls, the internalBoundary is in m (while the boundary from the backend in mm)